backend/app/services/klab/run.rb

```diff
@@ -0,0 +1,26 @@
+module Klab
+  class Run
+    def call
+      client = Klab::APIClient.new
+      # TODO: geometry
+      geometry = "aries.heco.locations.colombia_continental"
+      context_req = Klab::SubmitContextRequest.new(client)
+      context_resp = Klab::SubmitContextResponse.new(context_req.call(geometry))
+      ticket_req = Klab::TicketRequest.new(client, context_resp.ticket_id)
+      ticket_resp = Klab::TicketResponse.new(ticket_req.call)
+      # this part can take a few minutes - ticket gets resolved when observations for all indicators completed
+      until ticket_resp.resolved?
+        ticket_resp = Klab::TicketResponse.new(ticket_req.call)
+        sleep(5)
```

Comment:
Maybe it would be better to have background job to actually collect results, but sleep is probably fine for now

Reply:
yes, I guess a polling job every few seconds 👍